.env

```diff
@@ -1,2 +1,3 @@
 `PORT=5000`
-MONGODB_CONNECTION_STRING=mongodb://localhost:27017/SnelMelder
\ No newline at end of file
+MONGODB_CONNECTION_STRING=mongodb://localhost:27017/SnelMelder
+INDEVELOPMENT=false
```

Comment:
Please update the .gitignore this file shouldn't be here

Reply:
Removed